sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q38/explain.txt

```diff
@@ -1,54 +1,51 @@
 == Physical Plan ==
```

Comment:
can you summarize the plan changes? do we have less shuffles now?

Reply:
q14a/a14b reduces 1 Exchange and 2 HashAggregates.
q38 reduces 3 Exchange and 4 HashAggregates.
q38 reduces 3 Exchange and 4 HashAggregates.